MaxText/configs/v5p/llama2_7b.sh

```diff
@@ -0,0 +1,50 @@
+# Llama2 7B model.
+# The batch size is set to achieve 4M token global batch size on 1 x v5p-512.
+# If running on a different slice size, modify the per_device_batch_size field
+# accordingly to achieve desired batch size.
+#
+# Command Flags:
+# OUTPUT_PATH (Required, unless base_output_directory is already set in base.yml)
+# DATASET_PATH (Required, unless dataset_path is already set in base.yml)
+# RUN_NAME (Required, unless run_name is already set in base.yml or running with XPK/GKE)
+#
+# Example to invoke this script:
+# bash MaxText/configs/v5p/llama2_7b.sh RUN_NAME="<your_run_name>" OUTPUT_PATH="gs://<your_output_path>" DATASET_PATH="gs://<your_dataset_path>"
+#
+# Example to AOT compile:
+# bash MaxText/configs/v5p/llama2_7b.sh EXECUTABLE=train_compile.py M_COMPILE_TOPOLOGY=v5p-512 M_COMPILE_TOPOLOGY_NUM_SLICES=1
+
+
+# Stop execution if any command exits with error
+set -e
+
+export EXECUTABLE="train.py" # or train_compile.py
+export DATASET_TYPE="tfds"
```

Comment:
If `DATASET_PATH` is passed, can we set DATASET_PATH='' and DATASET_TYPE='synthetic'?

Reply:
Set dataset type to synthetic and dataset path shouldn't be needed by default